src/ncp/ncp_base_mtd.cpp

```diff
@@ -2619,6 +2619,118 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_THREAD_NETWORK_TIME>(
 }
 #endif // OPENTHREAD_CONFIG_ENABLE_TIME_SYNC
 
+#if OPENTHREAD_CONFIG_ENABLE_TIME_SYNC && OPENTHREAD_CONFIG_ENABLE_PERFORMANCE_TEST
+template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_PERFORMANCE_LATENCY_TEST>(void)
+{
+    otError             error = OT_ERROR_NONE;
+    const otIp6Address *peerAddr;
+    uint16_t            length;
+    bool                isSender;
+    otMessage *         message = NULL;
+    otMessageInfo       messageInfo;
+    char                payload[1500];
+
+    SuccessOrExit(error = mDecoder.ReadIp6Address(peerAddr));
+    SuccessOrExit(error = mDecoder.ReadUint16(length));
+    SuccessOrExit(error = mDecoder.ReadBool(isSender));
+
+    memset(&mSocket, 0, sizeof(mSocket));
+
+    error = otUdpClose(&mSocket);
+    SuccessOrExit(error);
+
+    error = otUdpOpen(mInstance, &mSocket, HandleUdpReceive, this);
+    SuccessOrExit(error);
```

Comment:
nit: combine `SuccessOrExit` line with previous expression? 
```
SuccessOrExit(error = otUdpOpen(mInstance, &mSocket, HandleUdpReceive, this))
```
Also in commands below?